cli_tools/osconfig_agent/ospackage/ospackage_linux.go

```diff
@@ -72,39 +72,147 @@ func SetConfig(res *osconfigpb.LookupConfigsResponse) error {
 	return errors.New(strings.Join(errs, ",\n"))
 }
 
-func aptRepositories(repos []*osconfigpb.AptRepository) error { return nil }
+func aptRepositories(repos []*osconfigpb.AptRepository) error {
+	/*
+		# Repo file managed by Google OSConfig agent
+		deb http://repo1-url/ repo1 main
+		deb http://repo1-url/ repo2 main contrib non-free
+	*/
+	var buf bytes.Buffer
+	buf.WriteString("# Repo file managed by Google OSConfig agent\n")
+	for _, repo := range repos {
+		line := fmt.Sprintf("\ndeb %s %s", repo.Uri, repo.Distribution)
+		for _, c := range repo.Components {
+			line = fmt.Sprintf("%s %s", line, c)
+		}
+		buf.WriteString(line + "\n")
+	}
+
+	return ioutil.WriteFile(config.YumRepoFilePath(), buf.Bytes(), 0600)
```

Comment:
Would these repository writers be cleaner if they used templates?

Reply:
Probably, added a TODO